admin/jobs/river/deployments_health_check.go

```diff
@@ -227,7 +238,7 @@ func (w *Worker) deploymentHealthCheck(ctx context.Context, d *database.Deployme
 	return instances, true
 }
 
-func (w *Worker) annotationsForDeployment(ctx context.Context, d *database.Deployment) (*admin.DeploymentAnnotations, error) {
+func annotationsForDeployment(ctx context.Context, w *DeploymentsHealthCheckWorker, d *database.Deployment) (*admin.DeploymentAnnotations, error) {
```

Comment:
Keep it as a member function since it's scoped to this type
```suggestion
func (w *DeploymentsHealthCheckWorker) annotationsForDeployment(ctx context.Context, w *DeploymentsHealthCheckWorker, d *database.Deployment) (*admin.DeploymentAnnotations, error) {
```